src/plugin.py

```diff
@@ -30,6 +30,28 @@
 
 
 class BNetPlugin(Plugin):
+    async def get_game_time(self, game_id, context):
```

Comment:
please move this from the top, e.g. just under get_local_games its good to have init first :)